src/server/handlers/launchpad.js

```diff
@@ -301,3 +301,57 @@ export const completeSnapAuthorization = async (req, res) => {
     });
   });
 };
+
+export const getSnapBuilds = (req, res) => {
+  const snap_link = req.query.snap_link;
+
+  if (snap_link) {
+    return getLaunchpad().get(snap_link).then(snap => {
+      // TODO: bartaz: currently just limiting to 10 recent builds
+      // we will need to process it properly later for paging/loading more
+      return getLaunchpad().get(snap.builds_collection_link, { start: 0, size: 10 })
```

Comment:
Currently just fetching 10 latest builds from LP.

Eventually we need to properly handle loading/paging here.